### PR TITLE
JS Capstone - API-based Webapp

### PR DESCRIPTION
**Pull request by omarbabou**
**The JavaScript capstone project is about building a web application based on the external TVmaze API (data about TV series and movies) with two user interfaces. For work coordination was used Kanban board with filled-in tasks. In development process was implemented:**

  - JavaScript to make websites dynamic and build basic single-page apps;
  - ES6 syntax: modules, callbacks, promises, etc.;
  - Webpack;
  - Jest for unit tests;
  - Gitflow;
  - Sending and receiving data from an API.

### CODE REVIEW

File: api.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  document.body.innerHTML = '<h1 class="count-movies"></h1>';
+  countMovies([1, 2, 3, 4, 5, 6, 7, 59, 89, 20]);
+  expect(document.querySelector('.count-movies').innerHTML).toBe('10');
+});
+
+test('displays nothing if there are no items on home-page', () => {
+  document.body.innerText = '<h1 class="count-movies"></h1>';
+  countMovies([]);
+  expect(document.querySelector('.count-movies').innerHTML).toBe('');
+});
```
Comment (julie-ify) [inline api.test.js]: - [x] These tests failed. I attached an image of the failing tests. Kindly fix it.
chrome_gRAyyGhLH5

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @omarbabou,

Good job so far!

## Highlights!
- GitFlow ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments here also 👇 ._

- [x] **[REQUIRED]** Kindly record a video presentation using the following guidelines:

- Recorded a video presenting the features of the project.
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description of the project in professional English. 
- The topic and purpose of the project are made clear.
- The video contains a highlight about a piece of the code you find interesting or is proud of making.


Note: It's recommended that you use [Zoom](https://zoom.us/) recording features.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/interact.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+export const addLike = async (itemId) => {
+  fetch(`${baseURL}apps/${appId}/likes`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ item_id: itemId }),
+  });
+};
```
Comment (BenMukebo) [inline src/modules/interact.js:12]: - [x] The counters functions should get covered with unit tests using [the Jest Framwework](https://jestjs.io/). Kindly add this, Here is the error message I got after I runing the tests in my computer 👇🏻 Kindly address this.

scree24

**Review by BenMukebo — CHANGES_REQUESTED** [review]
Hi @omarbabou, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

You have submitted a project, that Linters check are not running in the GitHub/action
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please Tag me on (@BenMukebo) when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @omarbabou, 

All tests passed 💯 Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**